docs/practices/when_to_use_private_repository.md

```diff
@@ -0,0 +1,37 @@
+# When to use private Github repository
+
+<blockquote style="background: rgba(255, 200, 0, 0.1); border: 5px solid rgba(255, 200, 0, 0.4);">
+
+The intended audience of the information presented here is developers working
+on the implementation of NEAR.
+
+Are you a security researcher? Please report security vulnerabilities to
+[security@near.org](mailto:security@near.org).
+
+</blockquote>
+
+## TL;DR; 
+
+Limit usage of nearcore-private to sensitive issues that need to be kept confidential
+
+
+## Details
+
+By nature, NEAR protocol ecosystem is community driven effort, welcoming everyone to make contribution; we believe in power of community and openness and bring a greater good. 
+In this regard, we encourage contributors to create an issue/PR in a public space, such as [nearcore](https://github.com/near/nearcore).
```

Comment:
I feel like this should be reworded so that we (as in Pagoda) do not come off as the owners of nearcore but one of the contributors.  Essentially, I would discourage using phrases like "we encourage ..." which makes it appear more like owners and less like contributors.

I know that today we are effectively the owners but we should be treating the repo as if we are just one of the contributors.